node/popn.go

```diff
@@ -433,6 +376,32 @@ func (nd *node) ping(ctx context.Context, pi peer.AddrInfo) error {
 	}
 }
 
+func (nd *node) peerInfo(ctx context.Context, addr address.Address) (*peer.AddrInfo, error) {
```

Comment:
Since we're getting info of a miner and not a pop peer, wouldn't it be better to name the function `minerInfo()`, to make the distinction between pop peers & filecoin miners ?

Reply:
Good point. Will update. Thanks